ocs_ci/ocs/ui/validation_ui.py

```diff
@@ -213,11 +213,12 @@ def odf_console_plugin_check(self):
         if default_projects_is_checked.get_attribute("data-checked-state") == "false":
             logger.info("Show default projects")
             self.do_click(self.validation_loc["show-default-projects"])
-        logger.info("Search for 'openshift-storage' project")
+        logger.info("Search for 'cluster_namespace' project")
```

Comment:
what about expanding the `cluster_namespace` here for the log message?
```suggestion
        logger.info("Search for '%s' project", config.ENV_DATA["cluster_namespace"])
```